Source/Core/UpdaterCommon/UpdaterCommon.cpp

```diff
@@ -375,7 +375,7 @@ void CleanUpTempDir(const std::string& temp_dir, const TodoList& todo)
 bool BackupFile(const std::string& path)
 {
   std::string backup_path = path + ".bak";
-  fprintf(log_fp, "Backing up unknown pre-existing %s to .bak.\n", path.c_str());
+  fprintf(log_fp, "Backing up %s to .bak.\n", path.c_str());
```

Comment:
Could we have it say `existing %s`, to make it clearer that it's about the old version and not the new version?